### PR TITLE
[14.0][MIG] edi_purchase_oca

### PR DESCRIPTION
**Pull request by sebalix**

### CODE REVIEW

**Comment by sebalix** [comment]
`pre-commit` doesn't want to work today:
```
An error has occurred: InvalidManifestError: 
==> File /home/salix/.cache/pre-commit/repowdxye_tq/.pre-commit-hooks.yaml
==> At Hook(id='black-jupyter')
==> At key: types_or
==> At index 2
=====> Type tag 'jupyter' is not recognized.  Try upgrading identify and pre-commit?
Check the log at /home/salix/.cache/pre-commit/pre-commit.log
```
But I can't find any `black-jupyter` hook in the current `.pre-commit-config.yaml` file...

EDI: I removed manually this `black-jupyter` hook from `$HOME/.cache/pre-commit/repowdxye_tq/.pre-commit-hooks.yaml` and now `pre-commit` is working. Will see later if this issue will pop-up again in another repo...

File: edi_purchase_oca/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+{
+    "name": "EDI Purchase",
+    "summary": """
+        Define EDI Configuration for Purchase Orders""",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi",
+    "depends": ["purchase", "edi_oca", "component_event"],
```
Comment (rousseldenis) [inline edi_purchase_oca/__manifest__.py:12]: @sebalix @LoisRForgeFlow I'm wondering why pulling the big edi_oca module as the code inside this does not depend on it ?

Comment (simahawk) [reply edi_purchase_oca/__manifest__.py:12]: that's wrong, see the model :wink: 
```
    _inherit = ["purchase.order", "edi.exchange.consumer.mixin"]
```

Comment (rousseldenis) [reply edi_purchase_oca/__manifest__.py:12]: Ouuuh, ok.

**Review by rousseldenis — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**